### DISCUSSION TITLE
Why do I see a black screen for some time?

### DISCUSSION THREAD

**Discussion by RahulMachhiC** [vestrel00/contacts-android#240] [Q&A] [answered]
Black Screen For Some time in android 11 than contacts load

**Comment by vestrel00** [comment, marked as answer]
@RahulMachhiC, this sounds like you are executing work in the UI thread instead of in a background thread.

Could you share some code so I can see it? Also, please read this page if you have not yet done so; https://vestrel00.github.io/contacts-android/async/async-execution-coroutines/

---------

The other possibility is that you are seeing a crash, in which case please check the error logs and share it here 🙏
